recipes/acts/recipe.yaml

```
@@ -0,0 +1,47 @@
+context:
+  name: acts
+  version: "41.1.0"
+
+package:
+  name: ${{ name|lower }}-core
+  version: ${{ version }}
+
+source:
+  git: https://github.com/acts-project/acts.git
+  tag: v${{ version }}
+
+build:
+  script:
+    - cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
+    - cmake -LH build
+    - cmake --build build --parallel="${CPU_COUNT}"
+    - cmake --install build
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+  host:
+    - libboost-devel
+    - eigen
+
+tests:
+  # Some packages might need a `test/commands` key to check CLI.
+  - script:
+    - echo "Completed build"
+
+about:
+  homepage: https://acts.readthedocs.io/en/latest/index.html
+  summary: 'ACTS is an experiment-independent toolkit for (charged) particle track reconstruction in (high energy) physics experiments implemented in modern C++.'
+  description: |
+    The ACTS project provides high-level track reconstruction modules that can be used for any tracking detector. The tracking detector geometry description is optimized for efficient navigation and fast extrapolation of tracks. Converters for several common geometry description packages are available. In addition to the algorithmic code, this project also provides an event data model for the description of track parameters and measurements.
+  license: MPL-2.0
+  license_file: LICENSE
+  documentation: https://acts.readthedocs.io/en/latest/index.html
+  repository: https://github.com/acts-project/acts
+
+extra:
```

```suggestion
extra:
  feedstock-name: acts
```

as this is going to be a multi-output feedstock the feedstock name needs to be provided and not inferred.